### ISSUE TITLE
version mismatch

### ISSUE THREAD

**Issue by cjyar** [IronCoreLabs/recrypt-wasm-binding#88]
[This build](https://github.com/IronCoreLabs/recrypt-wasm-binding/runs/6282615886?check_suite_focus=true) failed in the `Get version` step:
```
Version 'null' in './examples/create-react-app-example/package.json' doesn't match '0.6.24-pre' from others in './Cargo.toml
./examples/create-react-app-example/package.json
./package.json'
```

The example app has a version that doesn't match the other versions in the repo. The bump-version workflow doesn't know what to do with that.

The current logic for `package.json` files is: If `package.json` is in the root of the repo, read the version from it. Otherwise, ignore the deeply nested `package.json` and read the version from `package.json` in the root of the repo, which must exist.

Options:
- Change the example app so it has the same version as the repo.
- Nest the example app more deeply so it gets ignored by `find . -maxdepth 3`.
- Modify the bump-version scripts so they ignore that file. Maybe ignore deeply-nested `package.json` completely if there's no root `package.json`?

**Comment by cjyar** [comment]
@skeet70 suggests we just remove `version` from that file.

**Comment by cjyar** [comment]
I implemented that change in #89, but it was incorrect. Thanks to @BobWall23 for spotting the logic in the automation. Automation assumes that `examples/create-react-app-example/package.json` depends on `package.json` in the root. It reads `name` from the root, then searches for that name in `dependencies` in the example app. The logic is failing because the root has no `name` attribute.
